Source/Core/DolphinWX/ISOFile.cpp

```diff
@@ -287,6 +287,29 @@ bool GameListItem::ReadPNGBanner(const std::string& path)
   return true;
 }
 
+const std::string GameListItem::GetFileType() const
```

Comment:
The first const here is unnecessary. Ditto in the header, unless returning by reference.